mmdet3d/core/bbox/coders/km3d_decoder.py

```diff
@@ -0,0 +1,383 @@
+import numpy as np
+import torch
+from torch.nn import functional as F
+
+
+def _nms(heat, kernel=3):
+    """Process heat with Non Maximum Suppression.
+git
+    Args:
+        heat (torch.Tensor): Predictions from model in shape (batch_size *
+            channels * height * width)
+        kernel (int): kernel size for max pooling
+
+    Returns:
+        torch.Tensor (batch_size * channels * height * width): heatmap after
```

Comment:
The docstring format of return should be in [type]: description
```
"""
torch.Tensor: heatmap after NMS, in shape  (batch_size * channels * height * width)
"""
```

Reply:
See the docstring example here: https://sphinxcontrib-napoleon.readthedocs.io/en/latest/example_google.html